scout/server/blueprints/variants/controllers.py

```diff
@@ -888,6 +895,15 @@ def populate_filters_form(store, institute_obj, case_obj, user_obj, category, re
             }
         )
         clinical_filter = MultiDict(clinical_filter_dict)
+    elif category == "cancer_sv":
```

Comment:
Exactly: no way around that I fear since this will have both somatic stuff and sv stuff. Multiple inheritance might simplify it slightly, at least if this was c++.